### ISSUE TITLE
Chef problem with beetroot soup

### ISSUE THREAD

**Issue by Lleryn28** [TangoTek/TekTopia-Community#786]
**Describe the bug**
The chef will not make villager beetroot soup.  Instead she uses regular bowls, even though I keep collecting the bowls from the chests any time I see them.  She should only have access to villager oak wood and villager beetroot.  

**To Reproduce**
1. Go to 'chef'
2. Have 1 beetroot soup that you make somewhere on a villager or in a chest.
3. Chef then only makes non villager bowls and then non villager soup.

**Expected behavior**The chef should be able to make villager beetroot soup.

**Screenshots**

** Server/Location **
What server were you playing on? the Tektopia server
What are the world coordinates of your town hall marker?

**Additional context**

**Comment by Lleryn28** [comment]
Update to this:  I set the chef to no longer create beetroot soup or bowls and removed all bowls from circulation.  Once I removed all of the non villager bowls, then I started the chef again making her bowls.  She made villager bowls from that point on and had no trouble with the villager beetroot soup either.

**Comment by TangoTek** [comment]
So this seems to be working as intended?

**Comment by Lleryn28** [comment]
No, it seems as long as there is even just 1 bowl that is in circulation without the villager tag, all bowls made by the chef will end up with the non villager tag.  It should be that the chef can hold a non villager bowl and make just it into a non villager soup, but then the next bowl she makes should be a villager one.  Instead it also ends up being a non villager bowl.  I really hope this makes sense!


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________

**Comment by Lleryn28** [comment]
Update to this.  Ever since I solved the problem, it hasn't happened again.  I went to a single player world and tried to duplicate the problem and had no luck.  I am assuming it is fixed, and isn't an issue any more.
